code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -193,9 +193,9 @@
 	M.adjust_fire_stacks(1)
 	..()
 
-/datum/reagent/napalm/expose_mob(mob/living/M, method=TOUCH, reac_volume)
+/datum/reagent/napalm/expose_mob(mob/living/M, methods=TOUCH, reac_volume)
 	if(istype(M))
-		if(method != INGEST && method != INJECT)
+		if(methods & ~(INGEST|INJECT))
```

Comment:
```suggestion
		if(!(methods & (INGEST|INJECT)))
```

Reply:
Same thing and reasoning here. I could probably make it more explicit.